lms/templates/courseware/courseware.html

```diff
@@ -14,6 +15,11 @@
 </%def>
 
 <%block name="bodyclass">view-in-course view-courseware courseware ${course.css_class or ''}</%block>
+
+<%block name="js_overrides">
```

Comment:
@andy-armstrong This should work in main.html. I added this in courseware.html because it was only being used in courseware and i was unaware of the courseware-chromeless.html